### PR TITLE
Fix: Asc 21951 images in gallery sometimes show up and disappear

### PR DESCRIPTION
**Pull request by NaingAmity**

### CODE REVIEW

File: src/screens/UserProfile/Components/UserProfileGallery.tsx [hunk 1]
```diff
@@ -1,97 +1,117 @@
...
+          </TouchableOpacity>
+        );
+      },
+      [onPressThumbnail, styles.playButton, styles.thumbnail]
+    );
 
-  const renderGallery = useCallback(() => {
-    if (!data || data?.length === 0) {
-      const icon = tabName === TabName.Photos ? photo() : video();
+    const renderGallery = useCallback(() => {
```
Comment (bonanaaaaaa) [inline src/screens/UserProfile/Components/UserProfileGallery.tsx:63]: imo, we don't need `useCallback` here it's ok to use a condition rendering, wdyt?

File: src/screens/UserProfile/Components/UserProfileGallery.tsx [hunk 2]
```diff
@@ -1,97 +1,117 @@
...
-            <View style={styles.playButton}>
-              <SvgXml xml={playBtn} width="50" height="50" />
-            </View>
-          )}
-        </TouchableOpacity>
-      );
-    },
-    [onPressThumbnail, styles.playButton, styles.thumbnail]
-  );
+    const renderGalleryContent = useCallback(
```
Comment (bonanaaaaaa) [inline src/screens/UserProfile/Components/UserProfileGallery.tsx:47]: imo, we don't need `useCallback` here it's ok to use a condition rendering, wdyt?

File: src/screens/UserProfile/Components/UserProfileGallery.tsx (hunk 1 above)
Comment (NaingAmity) [reply src/screens/UserProfile/Components/UserProfileGallery.tsx]: Sure, removed!

File: src/screens/UserProfile/Components/UserProfileGallery.tsx (hunk 2 above)
Comment (NaingAmity) [reply src/screens/UserProfile/Components/UserProfileGallery.tsx]: Sure, removed!

**Review by bonanaaaaaa — APPROVED** [review]

**Pull request merged**